superset/commands/logs/prune.py

```diff
@@ -69,7 +69,7 @@ def run(self) -> None:
 
         total_rows = len(ids_to_delete)
 
-        logger.info("Total rows to be deleted: %s", total_rows)
+        logger.info("Total rows to be deleted: %s", f"{total_rows:,}")
```

Comment:
optional thought: it could be a good idea to have a `prettify_number` or similar to centralize this logic, with a few unit tests to verify/demonstrate how the value is formatted.